AnnoyancesFilter/sections/popups.txt

```diff
@@ -90,6 +90,7 @@ fforfree.net###popupFbLikeBox
 oneplus.com#$#.dialog-lock { overflow:visible!important; position:relative!important; top:0px!important; }
 oneplus.com#$##header-auto-subscribe { display:none!important; }
 15min.lt###feedback-btn
+anbariloche.com.ar##.contenedor_banner_popup
```

Comment:
It is an ad popup, please move.

Reply:
Please, tell me where exactly it should be?

Reply:
Spanish/specific